src/Retrier.php

```diff
@@ -14,24 +14,24 @@ public static function retryAsync(
         int $attempts,
         callable $action
     ): ExtendedPromiseInterface {
-        return new Promise(function (callable $resolve, callable $reject) use ($attempts, $action) {
+        return new Promise(static function (callable $resolve, callable $reject) use ($attempts, $action) {
             $retries = 0;
-            $shouldReject = function () use (&$retries, $attempts) {
+            $shouldReject = static function () use (&$retries, $attempts) {
                 return ++$retries >= $attempts;
             };
 
-            $executeAction = function (
+            $executeAction = static function (
                 callable $action
             ) use (
+                &$retries,
                 $resolve,
                 $shouldReject,
                 $reject,
-                &$executeAction,
-                &$retries
-            ) {
+                &$executeAction
+) {
```

Comment:
```suggestion
            ) {
```
I think this is the correct amount of spaces? hard to tell on GH

Reply:
Not sure about this, I've ran the formatter scripts via Composer before committing.